types/winston-syslog/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-var-requires": false
```

Comment:
Don't disable any tslint rules.

Reply:
I would love to not disable any rules, but I honestly don't know how to do `winston-syslog-tests.ts::3` without disabling a rule. Any suggestions?